src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/plugins/MonitorThreadContainer.java

```diff
@@ -46,28 +46,33 @@
     private final Map<IMonitor, Future<?>> tasksMap = new ConcurrentHashMap<>();
     private final Queue<IMonitor> availableMonitors = new ConcurrentLinkedDeque<>();
     private final ExecutorService threadPool;
+    private static final Object lockObject = new Object();
```

Comment:
https://stackoverflow.com/questions/1417190/should-a-static-final-logger-be-declared-in-upper-case